### ISSUE TITLE
C-c C-c after C-c C-z does nothing

### ISSUE THREAD

**Issue by Atrifex** [syl20bnr/spacemacs#7671]
I have this code currently in my .spacemacs file.
  (setq org-log-into-drawer "LOGBOOK")
  (setq org-clock-into-drawer 1)

I am trying to insert notes into logbooks. So to open the note I press C-c C-z. This seems to be working. However, when I press C-c C-c to save the note. Then for some reason, nothing happens.
Please let me know if I am doing something wrong.

**Comment by LemmingAvalanche** [comment]
What is your Org version? 9.0? Just asking more or less randomly since there seems to have been many issues in the wake of the new release.

**Comment by Atrifex** [comment]
My spacemacs version is 0.200.5 and my org version is 9.0.

**Comment by LemmingAvalanche** [comment]
Note that the documentation for the `org-log-into-drawer` variable says:

```
Do not check directly this variable in a Lisp program.  Call
function `org-log-into-drawer' instead.
```

So could you try using that function instead of using `setq` on the variable?

I just tried it and I got this message on startup:

```
(Spacemacs) Error in dotspacemacs/user-config: Symbol's function definition is void: org-log-into-drawer
```

So apparently this function stops existing when it tries to get called (that’s
the only thing I can surmise). Using `SPC h d f` I was able to lookup
`org-log-into-drawer` in the instance of Spacemacs where I _didn’t_ call this
function, but I was not able to look it up with `SPC h d f` in the instance of
Spacemacs where I got the above error. The same is true for the
identically-named variable `org-log-into-drawer`.

This has just left me more confused… 😁 

Anyone more knowledgable know what might be going on?

**Comment by TheBB** [comment]
You're calling `org-log-into-drawer` before org is loaded. Try doing `SPC hdf` in an Emacs where you haven't yet loaded org, you'll find no such function.

**Comment by LemmingAvalanche** [comment]
↑ Thank you! I’ll make some further tests with that in mind.

**Comment by Atrifex** [comment]
I am doing it after org is loaded. I am doing everything in a (with-eval-after-load 'org ...)  block of code and I am doing it in the config user-config(). I am not sure why but <C-c><C-c> still does nothing.

**Comment by LemmingAvalanche** [comment]
I tried using `with-eval-after-load` in my `dotspacemacs/user-config`:

``` elisp
(with-eval-after-load 'org
    ((org-log-into-drawer "LOGBOOK")
     (setq org-clock-into-drawer 1)))
```

And I’ll reiterate that the documentation for the `org-log-into-drawer` variable
_does_ say to use the `org-log-into-drawer` _function_

When I opened an Org file I got this error:

```
File mode specification error: (invalid-function (org-log-into-drawer "LOGBOOK"))
```

Even though I could look up the function using `SPC h d ` after having visited
the Org file.

Now TheBB will probably inform that I’ve made yet another trivial mistake
somewhere. 😁 So I think I’ll leave this question to someone more capable.

@Atrifex have you had any luck troubleshooting this? Do you now possess some
starting points, or something else that you can work from?

**Comment by Atrifex** [comment]
No I haven't. As a matter of fact, the changes I made produced another weird error.

No evertime I open a .org file. This line gets inserted at the start "te : CHANGED outside Customize.
   Non-nil means insert state change notes and time s" somewhere randomly into the file. A quick undo gets rid of it, but i would rather not have it happen.

**Comment by LemmingAvalanche** [comment]
Yeah, looking into this seemed to just uncover more and more weird (to me)
behaviour. ☺

**Comment by Atrifex** [comment]
Now I have gotten a more solvable issue I think. Now when I hit <C-c><C-z> it prints to the message log "Error in post-command-hook (org-add-log-note): (invalid-function org-babel-header-args-safe-fn)" I am still not sure how to solve this.

**Comment by Atrifex** [comment]
After a little bit of googling I figured out that the problems is that org and org-plus-contrib are both being downloaded by default. If you delete both of these folders and restart spacemacs, then it will download org-plus-contrib only and this will get rid on some conflict that exists within. 

For some reason, the first time I open a .org file that "te : CHANGED outside Customize.
Non-nil means insert state change notes and time s" is being pasted randomly somewhere in the file. I will make a separate issue for this since the current <C-c><C-c> issue has been solved.
